scheduler/src/cook/mesos/util.clj

```diff
@@ -167,22 +167,34 @@
 
 (timers/deftimer [cook-mesos scheduler get-pending-jobs-duration])
 
+(defn get-pending-job-ents*
```

Comment:
Should we make `get-pending-job-ents*` a private function? I assume the * in the name means that you want to encourage users to use `get-pending-job-ents`